support-frontend/assets/components/thankYou/signIn/signInItems.tsx

```diff
@@ -125,24 +142,18 @@ export function SignInBodyCopy({
 					</div>
 				</>
 			)}
-			{isTier3 && (
-				<>
-					<p>
-						<span>{upperCopyTier3}</span>
-					</p>
-				</>
-			)}
 		</>
 	);
 }
 
 export function SignInCTA({
 	email,
 	csrf,
-	isTier3,
+	isTierThree,
+	observerPrint,
 }: SignInBodyCopyProps): JSX.Element {
 	const [signInUrl, setSignInUrl] = useState('https://theguardian.com');
-
+	const isTierThreeOrObserver = observerPrint ?? isTierThree ?? false;
```

Comment:
```suggestion
	const isTierThreeOrObserver = observerPrint || isTierThree;
```

Reply:
Same linter issue here ->

![image](https://github.com/user-attachments/assets/98e9c228-1942-462b-87e9-2632ff2076d2)

Reply:
yes, our linter forces nullish coalescing. this is one of those cases where we def want to evaluate an empty string as falsy. we need to wrap it in a Boolean.
```suggestion
	const isTierThreeOrObserver = Boolean(observerPrint ?? isTierThree);
```

Reply:
probably this is one to discuss with the team. the `or` operator serves it purpose

Reply:
Perfect, perhaps we can go over this tomorrow morning. Thanks & Updated